dask/tests/test_imports_work.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+try:
+    import psutil
+except ImportError:
+    pytest.skip()
+
+
+def test_import_dask_dataframe(monkeypatch):
+    class MockOutput:
+        total = None
+
+    def mock_virtual_memory():
+        return MockOutput
+
+    monkeypatch.setattr(psutil, "virtual_memory", mock_virtual_memory)
+    import dask.dataframe  # noqa F401
```

Comment:
I suspect that at this point `dask.dataframe` may already be in `sys.modules` which would mean the mocking we're doing here won't have an impact. Instead we might do something like `assert _infer_block_size()` to ensure that an error isn't raised when `psutil.virtual_memory().total` is `None`

Reply:
Good point. This did fail, but I agree this will depend on the order pytest invokes the tests in. I will re-target the tests as you suggest.